common/dynamicconfig/constants.go

```diff
@@ -363,6 +363,9 @@ const (
 	QueuePendingTaskMaxCount = "history.queuePendingTasksMaxCount"
 	// QueueMaxReaderCount is the max number of readers in one multi-cursor queue
 	QueueMaxReaderCount = "history.queueMaxReaderCount"
+	// EnableWorkflowClosedCleanup to remove memo and search attributes from mutable state when workflow is closed
```

Comment:
If you want to have generic flag for closed workflows clean up, fix the comment to something like: 
```suggestion
	// EnableWorkflowClosedCleanup to clean up mutable state after workflow is closed such as remove memo and search attributes from it.
```
Or change the name of config key to `history.enableSearchAtributesAndMemoCleanUpOnClose`.